scripts/deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+export $(cat .env | xargs) &&
```

Comment:
I think this command would be more suited to the Makefile as it seems specific to developer environments.

Reply:
Ok removed this now  - you will need to remember to prepend any make commands with the notify key env var on your local dev environment

Reply:
or use a .env file